packages/frontend/src/view/MyAPI/Form/General.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Controller } from 'react-hook-form';
+
+import { FormGroup, Row, Col, Input } from '../../../ui';
+import { GeneralForm as Props } from '../../../types';
+import { StringArray } from '../../Form';
+import { useStyles } from './useStyles';
+
+export const GeneralForm: React.FC<Props> = ({
+  control,
+  errors,
+  addToken,
+  removeToken,
+  tokenFields,
+}) => {
+  const { formRow } = useStyles();
+  return (
+    <>
+      <FormGroup title="General settings">
+        <Row spacing={2}>
+          <Col xs={6}>
+            <Controller
+              as={Input}
+              name="name"
+              control={control}
+              label="Api name (required)"
```

Comment:
API is an abbreviation, should be all-caps.

Reply:
changed to `API ...`